src/pages/getting-started/installing-livechat/android-widget/index.mdx

```diff
@@ -319,6 +324,59 @@ android: configChanges = "orientation|screenSize";
 
 The chat widget will handle the orientation change by itself.
 
+# Localisation
```

Comment:
This whole section was not in our docs. However, it looks like it was added to Szymon's README before. In the current PRs from him, there is `Migrating to versions...` section added. That is why I decided to add the whole `localisation` section to our docs.